04-oop-basic-intro-to-dom/1-column-chart/index.js

```diff
@@ -1,3 +1,107 @@
 export default class ColumnChart {
 
+  _chartHeight = 50;
+
+  get chartHeight() {
+    return this._chartHeight;
+  }
+
+  constructor(obj) {
+    if (obj === undefined) {
+      this.render();
+      return;
+    }
+
+    this.data = obj.data;
+    this.title = obj.label;
+    this.header = obj.value;
+    this.link = obj.link;
+
+    if (obj.formatHeading !== undefined) {
+      this.formatHeading = obj.formatHeading;
+    }
+    else {
+      this.formatHeading = format => `${format}`;
+    }
+
+    this.render();
+  }
+
+  getLink() {
+    if (this.link === undefined) {
+      return '';
+    }
+
+    return `<a class="column-chart__link" href="/${this.link}">View all</a>`
```

Comment:
`return this.link ? <a class="column-chart__link" href="/${this.link}">View all</a> : ''`